app/install/dcos.md

```diff
@@ -92,11 +96,45 @@ have basic knowledge of [DC/OS]({{ page.links.dcos }}),
     following requests: 
 
     ```bash
-    $ curl marathon-lb.maraton.mesos:10001
-    $ curl marathon-lb.maraton.mesos:10002
+    $ curl marathon-lb.marathon.mesos:10001
+    $ curl marathon-lb.marathon.mesos:10002
     ```
 
-7. **Using Kong**
+7. **Deploy an upstream server**
+
+    For this demo, we will use an [app]({{ page.links.hello-app }}) which
+    returns `Hello world` on port `8080`. Using the `my_app.json` file from the
+    kong-dist-dcos repo, deploy the app in the cluster which will act as a
+    backend server to process requests received from Kong:
+
+    ```bash
+    $ dcos marathon app add my_app.json
+    ```
+
+8. **Using Kong**
+    
+    Create an API on Kong:
+
+    ```bash
+    $ curl -i -X POST marathon-lb.marathon.mesos:10002/apis \
+    --data "name=myapp" \
+    --data "hosts=myapp.com" \
+    --data "upstream_url=http://myapp.marathon.l4lb.thisdcos.directory:8080"
```

Comment:
One last thing: all of our shell commands examples in this website use a 4 space indentation on `--data` and other `curl` arguments.